ipn/ipnstate/ipnstate.go

```diff
@@ -67,6 +67,16 @@ type Status struct {
 	User map[tailcfg.UserID]tailcfg.UserProfile
 }
 
+type NetworkLockStatus struct {
+	Enabled bool
+
+	// Head describes the AUM hash of the leaf AUM.
+	Head *[32]byte
```

Comment:
what does nil here mean?

Reply:
`Head` is nil if network lock is not enabled. Added a comment.